CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Tamr S3 Module Repo
 
+## v1.1.0 - July 8th 2021
+* Sets default value for `additional_tags`
```

Comment:
The default value for `additional_tags` didn't change, so we can remove this line.